Dockerfile

```diff
@@ -1,8 +1,34 @@
 FROM ubuntu:xenial
 
-RUN apt-get update && \
-    apt-get install -y curl && \
+# Install the JDK
+# We are going to need some 32 bit binaries because aapt requires it
+# file is need by the script that creates NDK toolchains
+ENV DEBIAN_FRONTEND noninteractive
+RUN dpkg --add-architecture i386 && \
```

Comment:
We also need to install Android NDK & SDK similar to https://github.com/realm/realm-java/blob/master/Dockerfile for Android